.prettierrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  semi: true, // Add semicolons at the end of statements
```

Comment:
I added this configuration to lower the amount of superficial diffs (I usually have single quotes and no trailing commas in my projects)

This way other contributors can default to your standards